src/V3FactoryOwner.sol

```diff
@@ -177,7 +177,9 @@ contract V3FactoryOwner {
   /// method.
   /// @return _amount0 The amount0 fees collected, returned by the pool's collectProtocol method.
   /// @return _amount1 The amount1 fees collected, returned by the pool's collectProtocol method.
-  /// @dev See docs on IUniswapV3PoolOwnerActions for more information on forwarded params.
+  /// @dev The `UniswapV3Pool` contract only allows claiming the total amount minus 1 wei or less
```

Comment:
a different idea is to annotate the `@param _amount0Requested` and `@param _amount1Requested` with "max value" suggestions (`UniswapV3Pool:protocolFees.token0 - 1`, `UniswapV3Pool:protocolFees.token1 - 1`). do you think it helps at all?

Reply:
Yes, I think so. Should we keep a dev note as well or remove it altogether?

Reply:
https://github.com/uniswapfoundation/UniStaker/pull/73/commits/2b0cf560a5f07178f9b7fee6443ac68b0232c391